pkg/catalogv2/helm/helm2.go

```diff
@@ -36,10 +38,13 @@ var (
 	}
 )
 
+// isHelm2 checks if the value of the owner key of the received map is equal to TILLER
```

Comment:
// isHelm2 checks if the value of the owner key of the received map is equal to TILLER
// every helm2 release object contains this label.